test/e2e/lib/framework/serverApps/intake.ts

```diff
@@ -53,7 +53,7 @@ function computeIntakeType(
   }
 
   let intakeType: IntakeType
-  const forwardUrl = new URL(ddforward)
+  const forwardUrl = new URL(ddforward, 'http://example.com')
   const endpoint = forwardUrl.pathname.split('/').pop()
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** if we only use new proxy, we could assume that ddforward is already the path, no?‏
```suggestion
  const endpoint = ddforward.split('/').pop()
```

Reply:
I thought about it but we also have parameters here, so it would be a bit more work.

Reply:
What do you mean?

Reply:
oh, I see, what about:
```suggestion
  // ddforward=/api/v2/rum?key=value
  const endpoint = ddforward.split(/[\/\?]/)[3]
```